packages/forms/src/Components/FileUpload.php

```diff
@@ -184,6 +193,11 @@ public function getImageResizeMode(): ?string
         return $this->evaluate($this->imageResizeMode);
     }
 
+    public function getImageResizeUpscale(): ?string
```

Comment:
because this doesnt look right?

Reply:
Sorry about that @danharrin... I change it to `bool`